source/docs/03-metadata.xml

```diff
@@ -900,15 +900,16 @@
                      <specDesc key="work"/>
                   </specList>
                </p>
-               <p>All the components of <gi scheme="MEI">work</gi> are optional, but they must occur in the following order:</p>
+               <p>All the components of <gi scheme="MEI">work</gi> are optional. Before MEI 6.0 they had to occur in the following order:</p>
```

Comment:
@riedde @pe-ro If I get it right, there is no strict order any more for MEI 6.0, right? Would be good to have short indication of that

Reply:
Is this not clear by the change I propose?

Reply:
Since this is the documentation *for* MEI 6.0, then there's no need to specify that it's "Before MEI 6.0..." -- it's sufficient just to state it outright. "They may occur in any order".

If people want the pre-6 documentation where this is different, they can use that.

Reply:
Yes, good point @ahankinson. I thought about this after I had already made the comment above, too, and I agree, since we follow the same approach in other chapters as well. 
The documentation for pre-MEI6 can be found in the documentation for MEI5 which reads: `... are optional, but they must occur in the following order:`

Reply:
Okay, I just didn't want wo loose this information. But I'm absolutely with you @ahankinson @musicEnfanthen !